### PR TITLE
Add DEFAULT_UNIT setting

### PR DESCRIPTION
**Pull request by dnohales**
This is the first part of #92.

### CODE REVIEW

File: saas/static/js/djaodjin-saas.js [hunk 1]
```diff
@@ -385,7 +385,8 @@ CartItem.prototype = {
     };
 
     $.fn.invoice.defaults = {
-        currency_unit: "usd",
+        currency_unit: !globalSettings.default_unit ?
```
Comment (smirolo) [inline saas/static/js/djaodjin-saas.js]: It is OK to leave "usd" here and pass the default_unit when the jQuery component is initialized in the template.

        cardForm.find("#invoicables").invoice(
            {currency_unit: '{{lines_unit}}',
             saas_api_cart: "{{ urls.api_cart }}" });

lines_unit will always be initialized correctly by definition of how the template is constructed in the View.

File: saas/templates/saas/base_dashboard.html [hunk 2]
```diff
@@ -25,62 +89,7 @@
     $interpolateProvider.startSymbol('[[');
     $interpolateProvider.endSymbol(']]');
 }]);
-reopenSaasApp.constant('settings', {
```
Comment (smirolo) [inline saas/templates/saas/base_dashboard.html]: The whole point of doing ``reopenSaasApp.constant('settings', {...`` is that we do not rely on a global variables. Globals in Javascript, especially in libraries, create lots of integration problems down the line.

File: saas/backends/fake_processor.py [hunk 3]
```diff
@@ -22,9 +22,11 @@
 # OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
 # ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-import datetime, logging
+import datetime
+import logging
```
Comment (smirolo) [inline saas/backends/fake_processor.py]: Any reason to break those imports into two lines?

Comment (dnohales) [inline saas/backends/fake_processor.py]: Just a little disturbing PEP-8 warning ("E401: multiple imports on one line"). I suppose I need some configuration of what you use for linting.

File: saas/static/js/djaodjin-saas.js (hunk 1 above)
Comment (dnohales) [inline saas/static/js/djaodjin-saas.js]: Ok.

File: saas/templates/saas/base_dashboard.html (hunk 2 above)
Comment (dnohales) [inline saas/templates/saas/base_dashboard.html]: Yeah, it's quite ugly, but we can get rid of it as per the previous comment you made.

**Comment by dnohales** [comment]
Rebased and updated.

**Pull request merged**